### DISCUSSION TITLE
Module 'cloud_firestore' not found in GeneratedPluginRegistrant.h

### DISCUSSION THREAD

**Discussion by faustokeiluhu** [firebase/firebase-ios-sdk#8409] [Q&A]
Hi guys, I'm developing a project with flutter and I was trying to connect my app to the Cloud Firestore service that Firebase has.

I'm using VSCode as my primary IDE, but since I encountered this error when trying to build from terminal, I tried to build on xcode.

I use Xcode 12.5.1 and my machine is Macbook Pro 13 M1 Chip

I open XCode using Rosetta... I fI open Xcode without rosetta, the error is going to be Firebase Core module could not be found. So I tried opening using Rosetta.

Upon compiling the project, i've encountered this error : Module 'cloud_firestore' not found and it was labeled as a parse issue
<img width="965" alt="Error - Module Cloud Firestore not found" src="https://user-images.githubusercontent.com/87527398/126129123-91a3ef8e-64ed-428d-81db-a56864c37272.png">

I've tried several things
1. Add the GoogleService-Info.plist to the target runner
2. Cleaning the build folder and build the project again
3. Install the FirebaseFirestore dependencies from the git repository 
4. Use `use_frameworks! :linkage => :static` in my podfile

But all seems to produce the same error.

Here's my Build log
<img width="672" alt="Build Log Flutter" src="https://user-images.githubusercontent.com/87527398/126127626-fd871356-9a6b-4288-b5c0-744ae230b6cb.png">

Here's my `flutter doctor -v` result
<img width="602" alt="Flutter Doctor -v" src="https://user-images.githubusercontent.com/87527398/126128249-7a960a1d-1f52-4e57-bdce-db091dfab57a.png">

Here's my podfile
<img width="673" alt="podfile" src="https://user-images.githubusercontent.com/87527398/126128488-e776ba81-7b92-4676-8530-f62d1641c040.png">

Does anyone has an idea why is this happening?
I've spent days trying to figure this out but not able to resolve anything.
Thank you in advance 🙏🏻

**Comment by EdisonAltamirano** [comment]
Did anyone solve this bug?
